lib/exchange/provider/thorchain_exchange.provider.dart

```diff
@@ -72,10 +77,15 @@ class ThorChainExchangeProvider extends ExchangeProvider {
     try {
       if (amount == 0) return 0.0;
 
+      final formattedAmount =
+          (from == CryptoCurrency.usdterc20 || from == CryptoCurrency.usdc) && amount < 200
+              ? 200.00
+              : amount;
```

Comment:
I would rather it fail, than fetching a wrong rate.
so for example, say the user entered 50 USDT, and ThorChain returned a better rate than ChangeNow, because it will calculate on 200 instead of 50, but when creating it will fail, so why, just let it fail and it will return 0

Reply:
I got the point, it's useful only for fetchLimits and can be removed here.